examples/justfile

```
@@ -167,18 +297,20 @@ papi-generate:
     npm run papi:generate
 
 # Setup all services using Docker for IPFS
-setup-services:
+# Parameters:
+#   runtime - Runtime to build and run (e.g., "bulletin-polkadot-runtime", "bulletin-westend-runtime")
+setup-services runtime="bulletin-polkadot-runtime":
     #!/usr/bin/env bash
     set -e
     
     echo "🔧 Tearing down Docker services if they are running..."
     just ipfs-shutdown
     
-    echo "🐳 Setting up services with Docker IPFS..."
+    echo "🐳 Setting up services with Docker IPFS (runtime: {{ runtime }})..."
     
     # Start services with Docker
     just ipfs-start
-    just bulletin-solo-zombienet-start
+    just bulletin-solo-zombienet-start {{ runtime }}
```

And `network` or `network-setup` could be "solo" or "para" value specified in runtimes-matrix.json

```suggestion
    if [ "{{ network }}" = "solo" ]; then
        just bulletin-solo-zombienet-start {{ runtime }}
    if [ "{{ network }}" = "para" ]; then
        just bulletin-para-zombienet-start {{ runtime }}    
    fi        
```

---

> also the zombienet file ( /zombienet/bulletin-westend-local.toml or solo) should be specified in runtimes-matrix.json

I am not sure if you mean File name or Path to a File? I think Path needs to be relative because working directory may differ. I'd leave as is (`ZOMBIENET_CONFIG=$(cd .. && pwd)/zombienet/bulletin-westend-local.toml`)

> and also chain spec command could be as a part of runtimes-matrix.json (./scripts/create_bulletin_westend_spec.sh)

Working directory is `examples` so path needs to be `.././scripts/create_bulletin_westend_spec.sh`.

What I would do, I would decouple `PATH` from `filename` and handle them separately:
- PATH stays relative as is in the `justfile`
- add two new properties to `runtimes_matrix`: `zombienet_file_name` and `chain_spec_script_name`
This way we can have both relative path and configurable file names

WDYT?